test/AssertionsTests.cpp

```diff
@@ -1,3 +1,4 @@
+#define CATCH_CONFIG_FAST_COMPILE
```

Comment:
Can we not set this in the build configuration instead? This feels very boilerplatey.

Reply:
Sure, I was considering that, but figured it will be quicker for me to just add it in.

Do you want this to be an `option()`? Default on?

I think I'll take this commit out and resubmit later in a separate PR.